### ISSUE TITLE
multiauth translation broken

### ISSUE THREAD

**Issue by chromepenguin** [simplesamlphp/simplesamlphp#1206]
**Describe the bug**
When using multiauth module as an authentication source, "no translation" is displayed regardless of what is specified in the 'text' attribute or even if the default authsource name is used.

For example, when using the multisource module with the following text set:
```
'multisource' => [
    'multiauth:MultiAuth',
    'sources' => [
        'example-userpass' => [
            'text' => [
                'en' => 'Example Login',
          ]
    ]
]
```

I would expect it to display "Example Login" as the button text. However it displayed "not translated (Example Login)" instead.

**To Reproduce**
1. Enable multiauth:MultiAuth in config/authsource.php
2. Add an auth source with or without text
3. Attempt to log in

**Expected behavior**
Display text in the language as specified, falling back to the default language.

**Screenshots or logs**
![image](https://user-images.githubusercontent.com/1079669/65547802-2c636080-dee8-11e9-822a-4e5f28204a01.png)

**Additional context**
SimpleSamlPHP version 1.17.6

**Comment by tvdijen** [comment]
Reproducible with 1.18 / master as well.. Affects old UI only

**Comment by chromepenguin** [comment]
I created a PR with a possible solution, but perhaps I'm missing something here and something else needs to be done. I am upgrading from 1.13 to 1.17 using multiauth and it works fine in the old version.

**Comment by tvdijen** [comment]
That's actually funny, because the call to $this->t() has been there like since forever...

**Comment by tvdijen** [comment]
Thanks for the PR, I've merged it and it will be part of 1.18 which will be released soon
